### PR TITLE
Moves paramedic clothes from the locker into the medidrobe

### PR DESCRIPTION
**Pull request by Moltijoe**
# Why is this good for the game?
because this is a little ridiculous
image

# Testing
no need with how simple this change is, but i can prove it if need be

:cl:  
bugfix: fixed a bugged suit being given to paramedics
tweak: Moves paramedic clothes from the locker into the medidrobe
/:cl:

### CODE REVIEW

**Comment by ktlwjec0** [comment]
I will upvote, but I'd personally just up the count of these in the medidrobe, since the dufflebags are really only used when they don't have access to a vendor. Like the hop or hos.

**Comment by Moltijoe** [comment]
> I will upvote, but I'd personally just up the count of these in the medidrobe, since the dufflebags are really only used when they don't have access to a vendor. Like the hop or hos.

Ah right

**Review by neeshacark — APPROVED** [review]
no issues with maps, this still needs a stinky coder maint review

File: code/modules/vending/wardrobes.dm [hunk 1]
```diff
@@ -79,26 +79,30 @@
 	vend_reply = "Thank you for using the MediDrobe!"
 	products = list(/obj/item/clothing/accessory/pocketprotector = 4,
 					/obj/item/storage/backpack/duffelbag/med = 4,
-					/obj/item/storage/backpack/medic = 4,
```
Comment (ynot01) [inline code/modules/vending/wardrobes.dm:82]: why?

**Review by ynot01 — APPROVED** [review]
yesd

**Pull request merged**